src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -80,6 +80,7 @@ class SentryAppSerializer(Serializer):
     overview = serializers.CharField(required=False, allow_null=True)
     verifyInstall = serializers.BooleanField(required=False, default=True)
     allowedOrigins = ListField(child=serializers.CharField(max_length=255), required=False)
+    popularity = serializers.IntegerField(min_value=0, max_value=32767, required=False)
```

Comment:
Where does `max_value` come from? Like why pick 32767?

Reply:
The field we use to store it on the model! https://docs.djangoproject.com/en/3.2/ref/models/fields/#positivesmallintegerfield